terragrunt/modules/crates-io/cloudfront-static.tf

```diff
@@ -45,6 +63,12 @@ resource "aws_cloudfront_distribution" "static" {
         forward = "none"
       }
     }
+
+    lambda_function_association {
```

Comment:
Based on last ~30 days I think this would mean 339 million * $0.60/million requests - so roughly an extra ~$200/month. I think if we can switch to cloudfront functions we get 6x cheaper pricing ($0.10/million requests) and presumably roughly the same functionality for the redirects desired here?

Reply:
Great suggestion! I refactored the code to use a CloudFront function instead.